### ISSUE TITLE
Edge portals are misaligned

### ISSUE THREAD

**Issue by cyu06** [palantir/blueprint#3735]
#### Environment

- __Browser and OS versions__: Windows MS Edge

#### Steps to reproduce

1. Render a modal with MS Edge

#### Actual behavior

Modals are occasionally offset by a fixed distance

#### Expected behavior

Modals render where expected

#### Possible solution

We believe https://github.com/palantir/blueprint/blob/06a186c/packages/core/src/components/portal/_portal.scss#L9 could possibly be updated; when the portal's position is changed to fixed instead of absolute the modal correctly shifts back to the expected position.

**Comment by invliD** [comment]
Specifically, render a dialog within an iframe, it may then be offset within the iframe by the iframe's offset (resulting in a double offset compared to the _window_)

**Comment by adidahiya** [comment]
You're right in pointing out that code might be outdated since we're not using Tether anymore... but I can't seem to repro this in the code sandbox on Edge on Mac or Windows (through Parallels). Can you repro it in the sandbox? Do you have a screenshot?

**Comment by adidahiya** [comment]
^ bump @cyu06 @invliD

**Comment by invliD** [comment]
I can't seem to repro in sandbox either, but I can send you a screenshot of one of our products internally.
